src/AgFEM/CellAggregation.jl

```diff
@@ -53,47 +55,61 @@ function aggregate(
   geo::CSG.Geometry,
   in_or_out)
 
+  trian = Triangulation(cut,geo)
+  cell_to_cut_meas = cell_measure(trian,num_cells(cut.bgmodel))
   cell_to_inoutcut = compute_bgcell_to_inoutcut(cut,geo)
   facet_to_inoutcut = compute_bgfacet_to_inoutcut(cut_facets,geo)
-  aggregate(strategy,cut.bgmodel,in_or_out,cell_to_inoutcut,facet_to_inoutcut)
+  aggregate(strategy,cut.bgmodel,in_or_out,cell_to_cut_meas,cell_to_inoutcut,facet_to_inoutcut)
 end
 
-function aggregate(strategy,model::DiscreteModel,in_or_out,cell_to_inoutcut,facet_to_inoutcut)
+function aggregate(
+  strategy,model::DiscreteModel,in_or_out,cell_to_cut_meas,cell_to_inoutcut,facet_to_inoutcut)
```

Comment:
I see that you have needed to modify the signature of this abstract method, i.e. the method that the different strategies need to implement.  It means that the design of the abstract interface was not good enough (my fault). Ideally, new aggregation strategies should not need to modify the signature of the abstract method (otherwise the code is not really extensible).

Perhaps it is a good moment to think about a signature that will be general enough to other aggregations strategies. We can discuss this face-to face.

Reply:
Done in dd3fc66